README.md

```diff
@@ -10,19 +10,90 @@ The aim of this project is to classify the income category people in a binary fa
+
+
+```python
+# we use lrp.z here out of various methods available like 
+# Saliency, Deconvnet, GuidedBackprop, SmoothGrad, and IntergratedGradient
+# then we pass our model here
+analyzer = innvestigate.create_analyzer("lrp.z", model)
+# then we pass model inputs, that is x_train
+analysis = analyzer.analyze(x_train)
+```
+Then we perform the layer wise layer analysis by blocking off the layers and then performing analysis on them. 
+
+```python
+# Here we cutoff the layers and define a new model. This will remove the last Dense layer.
+new_model_1 = Model(model.inputs, model.layers[-3].output)
+# model.get_weights() return a numpy list of weights
+new_model_1.set_weights(model.get_weights())
+# summarize the model
+new_model_1.summary()
+```
+Then we perform the analysis on this new model:
+
+```python
+analyzer = innvestigate.create_analyzer("lrp.z", new_model_1)
+analysis = analyzer.analyze(x_train)
+# We save the results as numpy vectors
+np.save("out_lrp", analysis)
+```
+We perform this method on every layers, meanwhile saving the numpy vectors.
+
+Using this, we get layer by layer heatmaps in form of numpy vectors. In summary, we are cutting off layers in the network to obtain heatmap vectors for every layer and we are then merging them together to form a heatmap matrix/tensor. We use lrp_matrix.py to get complete numpy vectors then we plot them.
+
+## Choosing Appropriate Model
+
+We perforemed experiments with various model architecture to choose the right one. We present a table below stating the model architecture and accuracy. 
+
+Dense layer : 'D', Batch Normalization: 'BN', Dropout: 'DR'
+<div align='center'>
+
+| Model Architecture  |  Train acurracy | test accuracy  |
+|:-:|:-:|:-:|
+| D512-SELU-D256-BN-SELU-D128-BN-SELU-D64-BN-SELU-D64-BN-SELU-D32-BN-SELU-D16-BN-SELU-D8-BN-SELU-D2-SIGMOID  |  79.64 | 42  |
+| D512-SELU-D512-BN-SELU-D128-BN-SELU-D64-BN-SELU-D64-BN-SELU-D32-BN-SELU-D16-BN-SELU-D8-BN-SELU-D2-SIGMOID  | 81.19  |  78.643 |
+|  D512-SELU-D512-BN-SELU-D128-BN-SELU-D128-BN-SELU-D64-BN-SELU-D32-BN-SELU-D16-BN-SELU-D8-BN-SELU-D2-SIGMOID |  80.37 |  59.787 |
+| D512-SELU-D512-BN-SELU-D512-BN-SELU-D128-BN-SELU-D64-BN-SELU-D32-BN-SELU-D16-BN-SELU-D8-BN-SELU-D2-SIGMOID  | 80.38  |  57.14 |
+| D256-RELU-D256-BN-RELU-D128-BN-RELU-D64-BN-RELU-D64-BN-RELU-D32-BN-RELU-D16-BN-RELU-D8-BN-RELU-D2-SIGMOID  |  80.20 | 77.25  |
+| D512-RELU-D512-BN-RELU-D256-BN-RELU-D64-BN-RELU-D64-BN-RELU-D32-BN-RELU-D16-BN-RELU-D8-BN-RELU-D2-SIGMOID  |  80.04 | 61.16  |
+| SAME PREVIOUS MODEL, EPOCHS=20000  | 92.477  | 78.84  |
+| D256-RELU-D256-BN-RELU-D128-BN-RELU-D128-BN-RELU-D64-BN-RELU-D64-BN-RELU-D32-BN-RELU-D16-BN-RELU-D8-BN-RELU-D2-SIGMOID  |  91.545 | 78.68  |
+|  D512-RELU-D512-BN-RELU-DR(0.4)-D256-BN-RELU-DR(0.4)-D256-BN-RELU-DR(0.4)-D128-BN-RELU-DR(0.3)-D128-BN-RELU-DR(0.3)-D64-BN-RELU-DR(0.3)-D64-BN-RELU-DR(0.3)-D64-BN-RELU-DR(0.3)-D64-BN-RELU-DR(0.3)-D8-BN-RELU-DR(0.3)-D2-SIGMOID, EPOCHS=30000 | 88.23  | 81.78  |
```

Comment:
@thevoxium Would you mind highlighting this line in bold face?

Reply:
Sorry but I can't understand which line to bold, like making the whole table bold??

Reply:
The line containing the score of the best model.